### PR TITLE
New urls

### PR DESCRIPTION
**Pull request by iuliachiriac**

### CODE REVIEW

**Comment by iuliachiriac** [comment]
@alexef please review

File: gemet/thesaurus/templates/concept.html [hunk 1]
```diff
@@ -77,7 +77,7 @@ <h6 class="dt">Broader</h6>
     <ul class="dd-list">
       {% for cp in concept.broader_concepts %}
       <li>
-        <a href="{% url 'concept' language.code cp.id %}">{{ cp.name }}</a></li>
+        <a href="{% url 'concept' language.code cp.concept__code %}">{{ cp.name }}</a></li>
```
Comment (alexef) [inline gemet/thesaurus/templates/concept.html]: Maybe we should make `concept__code` value a property on cp?

Comment (alexef) [inline gemet/thesaurus/templates/concept.html]: ... or that will mess up with query.values?

File: gemet/thesaurus/templates/downloads/backbone.rdf [hunk 2]
```diff
@@ -41,6 +41,11 @@
 {% for group in groups %}
     <skos:Collection rdf:about="group/{{ group.code }}">
         <rdf:type rdf:resource="{{ group_uri }}"/>
+        <gemet:subGroupOf>
+            <rdf:Description rdf:about="supergroup/{{ group.source_relations__target__code }}">
```
Comment (alexef) [inline gemet/thesaurus/templates/downloads/backbone.rdf]: ugly... can we property it? (`group.source_relations__target__code`)

File: gemet/thesaurus/templates/downloads/language_definitions.rdf [hunk 3]
```diff
@@ -9,8 +12,8 @@
 {% for concept_code, definition in definitions.items %}
     <rdf:Description rdf:about="concept/{{ concept_code }}">
     {% for d in definition %}
-    <skos:{{ d.name }}>{{ d.value }}</skos:{{ d.name }}>
+    <skos:{{ d.name }}>{{ d.value|normalize:"NFC" }}</skos:{{ d.name }}>
```
Comment (alexef) [inline gemet/thesaurus/templates/downloads/language_definitions.rdf]: nice

File: gemet/thesaurus/urls.py [hunk 4]
```diff
@@ -81,19 +81,19 @@
         url(r'^inspire-themes/$', InspireThemesView.as_view(),
             name='inspire_themes'),
         url(r'^groups/$', GroupsView.as_view(), name='groups'),
-        url(r'^concept/(?P<concept_id>\d+)$', TermView.as_view(),
+        url(r'^concept/(?P<code>\d+)$', TermView.as_view(),
```
Comment (alexef) [inline gemet/thesaurus/urls.py]: aren't there concepts with `[a-z]` concept codes??

File: gemet/thesaurus/urls.py [hunk 5]
```diff
@@ -81,19 +81,19 @@
...
             name='concept'),
-        url(r'^inspire-theme/(?P<concept_id>\d+)$',
+        url(r'^inspire-theme/(?P<code>[a-zA-Z]+)$',
```
Comment (alexef) [inline gemet/thesaurus/urls.py]: only these ones, right?

File: gemet/thesaurus/urls.py [hunk 6]
```diff
@@ -81,19 +81,19 @@
...
+        url(r'^theme/(?P<code>\d+)$', ThemeView.as_view(),
             name='theme'),
-        url(r'^group/(?P<concept_id>\d+)$', GroupView.as_view(),
+        url(r'^group/(?P<code>\d+)$', GroupView.as_view(),
             name='group'),
-        url(r'^supergroup/(?P<concept_id>\d+)$', SuperGroupView.as_view(),
+        url(r'^supergroup/(?P<code>\d+)$', SuperGroupView.as_view(),
             name='supergroup'),
-        url(r'^relations/(?P<group_id>\d+)/$', RelationsView.as_view(),
+        url(r'^relations/(?P<group_code>\d+)/$', RelationsView.as_view(),
```
Comment (alexef) [inline gemet/thesaurus/urls.py]: trailing `/`?

File: gemet/thesaurus/urls.py [hunk 7]
```diff
@@ -81,19 +81,19 @@
...
             name='relations'),
-        url(r'^theme/(?P<theme_id>\d+)/concepts/$',
+        url(r'^theme/(?P<theme_code>\d+)/concepts/$',
```
Comment (alexef) [inline gemet/thesaurus/urls.py]: also here?

File: gemet/thesaurus/urls.py (hunk 4 above)
Comment (iuliachiriac) [inline gemet/thesaurus/urls.py]: nope, only inspire themes have them

File: gemet/thesaurus/urls.py (hunk 6 above)
Comment (iuliachiriac) [inline gemet/thesaurus/urls.py]: The URLs which shouldn't contain a trailing `/`are those that are referenced from the downloadable files. For `relations` and `theme_concepts` the URL can stay the way it was.

**Pull request merged**